contracts/pcv/ERC20HoldingPCVDeposit.sol

```diff
@@ -0,0 +1,101 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.0;
```

Comment:
some of the contracts in this PR use `0.8.13` and some use `^0.8.0` any reason for this?